### PR TITLE
HSM Page

### PR DESCRIPTION
**Pull request by cindyzhang12**
## Summary
- need to add icons

### CODE REVIEW

File: src/components/pages/HSMPage/HSMPage.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    columnNames: columnNames,
+    columns: getColumns,
+    columnStyles: columnStyles,
+  };
+
+  return (
+    <List
+      title="HSM Messages"
+      listItem="sessionTemplates"
+      listItemName="speed send"
```
Comment (kurund) [inline src/components/pages/HSMPage/HSMPage.tsx:41]: This seems incorrect too. This should be "template". Please update the code when you copy+paste

File: src/App.tsx [hunk 2]
```diff
@@ -43,6 +44,9 @@ const App = () => {
             <Route path="/speed-send/add" exact component={MessageTemplate} />
             <Route path="/speed-send/:id/edit" exact component={MessageTemplate} />
             <Route path="/chat" exact component={Chat} />
+            <Route path="/templates" exact component={HSMPage} />
```
Comment (kurund) [inline src/App.tsx:47]: Please use singular so "template"

File: src/App.tsx [hunk 3]
```diff
@@ -43,6 +44,9 @@ const App = () => {
...
+            {/* This part isn't working properly */}
```
Comment (kurund) [inline src/App.tsx:48]: Why is this added? Please clean up unnecessary code.

Comment (cindyzhang12) [reply src/App.tsx:48]: That comment was there before, I wasn't sure who added it
